staging/src/k8s.io/kubelet/config/v1beta1/types.go

```diff
@@ -989,6 +989,14 @@ type KubeletConfiguration struct {
 	// +featureGate=GracefulNodeShutdown
 	// +optional
 	ShutdownGracePeriodCriticalPods metav1.Duration `json:"shutdownGracePeriodCriticalPods,omitempty"`
+	// ShutdownGracePeriodByPodPriority specifies the shutdown grace period for Pods based on their associated priority class value.
```

Comment:
Describe EXACTLY how this works in one sentence.  Possibly something like:

> When a shutdown request is received, the Kubelet will initiate shutdown on all pods running on the node with a grace period that depends on the priority of the pod, and then wait for all pods to exit.

Right now it's not clear whether this modifies the behavior of shutdown.